Mathlib/CategoryTheory/Limits/Shapes/Images.lean

```diff
@@ -261,6 +326,10 @@ section
 
 variable [HasImage f]
 
+/-- Some image factorisation of `f` through a monomorphism (selected with choice). -/
+def Image.imageFactorisation : ImageFactorisation f :=
+  Classical.choice HasImage.exists_image
+
```

Comment:
You should update the definitions below so that they use `Image.imageFactorisation`.

Reply:
Done!